src/edu/macalester/comp127/hw0/Area.java

```diff
@@ -21,7 +21,7 @@ public static double getSquareArea(double sideLength) {
      */
     public static double getCircleArea(double radius) {
         // TODO: replace me with the correct formula
-        return radius * 1.0;
+        return radius * radius * 3.1415926535897932384626433;

```

Comment:
Instead of typing out the digits of pi, you can use the following to be more exact:
``` return radius * radius * Math.PI```